conform-pr/action.yml

```diff
@@ -1,6 +1,6 @@
 ---
-name: 'PR verification'
-description: 'Verifies that PR`s title does not end with dot. Works for PR-related events only.'
+name: 'Conform PR'
+description: 'Checks that PR conforms to our process guides'
```

Comment:
https://www.merriam-webster.com/dictionary/conform

The `name` field uses "conform" as a transitive verb that feels a bit strange to me, but looks like it is grammatically correct

Reply:
The usage looks good to me. I'm not sure I'd use the verb "conform" as it sounds too official, like something compliance-related, but I don't have a strong opinion, so I'm fine with this verb.

What I _do_ miss in actions' descriptions/readme in this repo is why such an action exists and what are usage recommendations for them. Or where exactly such actions are used in FerretDB repos. But I believe it might be a separate PR with some README improvements.

Reply:
One additional benefit of "conform" is that it is shorter than "verification", so it is not cliped by GitHub UI